scene/animation/animation_mixer.cpp

```diff
@@ -631,10 +635,19 @@ bool AnimationMixer::_update_caches() {
 			if (!track) {
 				Ref<Resource> resource;
 				Vector<StringName> leftover_path;
-				Node *child = parent->get_node_and_resource(path, resource, leftover_path);
 
+				if (!parent->has_node_and_resource(path)) {
+					if (check_path) {
+						WARN_PRINT("AnimationMixer: '" + String(E) + "', couldn't resolve track:  '" + String(path) + "'. If there is no problem that the track value is not applied to the scene and you don't want further warnings, you can turn off the checking for the animation track path in Project Settings.");
```

Comment:
```suggestion
						WARN_PRINT("AnimationMixer: '" + String(E) + "', couldn't resolve track:  '" + String(path) + "'. This warning can be disabled in Project Settings.");
```
Less verbose. Though I'm not sure how important is the information you provided (I mean the "If there is no problem that the track value is not applied to the scene" part).